### DISCUSSION TITLE
Use of proxy server

### DISCUSSION THREAD

**Discussion by SumitNandi-eaton** [Azure/azure-c-shared-utility#610] [General]
I am trying to add proxy server to connect my zephyrRTOS based stm32f4 IoT edge device with Azure IoT hub. I am using MQTT_websockets, mbedTLS and network stack provide with zephyr RTOS.

Is there any example available to refer?

**Comment by ericwolz** [comment]
Here is a sample on setting the proxy.

https://github.com/Azure/azure-iot-sdk-c/blob/32942abb994f6855edc2a4dee8afd80735df4151/iothub_client/samples/iothub_convenience_sample/iothub_convenience_sample.c#L294
